Guesser/FilterTypeGuesser.php

```diff
@@ -60,6 +60,11 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                     return new TypeGuess('doctrine_orm_model', $options, Guess::HIGH_CONFIDENCE);
             }
         }
+        
+        if (!isset($metadata->fieldMappings[$propertyName]))
+        {
+            throw new \Exception('Property ' . $propertyName . ' not found in class' . $class . '!');
```

Comment:
Yes this is much better! 

@ju1ius I think you can post a new PR, I will close this one. Thanks you